site/app/controllers/grading/SimpleGraderController.php

```diff
@@ -124,6 +124,9 @@ public function gradePage($gradeable_id, $view = null, $sort = null) {
         elseif ($sort === "first") {
             $sort_key = "coalesce(NULLIF(u.user_preferred_firstname, ''), u.user_firstname)";
         }
+        elseif ($sort == "section_subsection") {
+            $sort_key = "u.registration_subsection";
+        }
         else {
             $sort_key = "coalesce(NULLIF(u.user_preferred_lastname, ''), u.user_lastname)";
         }
```

Comment:
If the intention is to make this the default sort method, than you should set it to be the last element in the `if/elseif/else` chain here. This also means you don't need to modify the `$sort` parameter.
```suggestion
        elseif ($sort === "last") {
            $sort_key = "coalesce(NULLIF(u.user_preferred_lastname, ''), u.user_lastname)";
        }
        else {
            $sort_key = "u.registration_subsection";
        }
```